src/rpdk/core/contract/suite/contract_asserts.py

```diff
@@ -95,7 +98,9 @@ def response_contains_resource_model_equal_updated_model(
     assert response["resourceModel"] == {
         **current_resource_model,
         **update_resource_model,
-    }
+    }, "All properties specified in the update request MUST be present in the \
+        model returned, and they MUST match exactly, with the exception of \
+            properties defined as writeOnlyProperties in the resource schema"
```

Comment:
think this error message might be misleading because there should be more exceptions:
[readOnlyProperties, writeOnlyProperties, defaults, insertionOrder, uniqueItems, etc.](https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/#resource-semantics)